reactive/jenkins.py

```diff
@@ -177,3 +198,29 @@ def migrate_charm_data():
 def stop():
     service_stop("jenkins")
     status_set("maintenance", "Jenkins stopped")
+
+
+@when("apt.installed.jenkins")
+@when("reset.jenkins.home")
+def reset_jenkins_home():
+    status_set("maintenance", "Configuring JENKINS_HOME")
+
+    home = unitdata.kv().get("storage.dir")
+    if home is None:
+        home = '/var/lib/jenkins'
+    log("Setting Jenkins home to {}".format(home))
+    unitdata.kv().set("jenkins.home", home)
+    # re-import paths to force update
+    importlib.reload(paths)
```

Comment:
That was a change I made also, https://github.com/jenkinsci/jenkins-charm/pull/18/files#diff-51fc11c4cd6e41d328a7e7d4f68cf801R8

The fact that this change is part of the diff just adds to the argument that these shouldn't be globals anymore.